components/cam/src/dynamics/se/dp_coupling.F90

```diff
@@ -1,279 +1,280 @@
-!
-!-------------------------------------------------------------------------------
+!---------------------------------------------------------------------------------------------------
 ! dynamics - physics coupling module
-!-------------------------------------------------------------------------------
+!---------------------------------------------------------------------------------------------------
```

Comment:
@whannah1, since quite a bit of the changes to this code are related to indenting and reordering the use module calls could you briefly respond to this comment with a list of the actual changes to the code that we can focus on?

Reply:
The only important parts here are where the the mapping between the GLL and FV grids occurs. Everything functions the same, like the conversion between chunks and blocks is the same. 

I had previously changed it a little bit to make certain variables allocatable, but Oksana pointed out that this wasnt a good idea so I reverted those changes.

Reply:
the "write_inithist" part is also probably important to look at.

Reply:
About what line numbers are the GLL and FV grid mappings?

Reply:
Basically, every section that's wrapped in "if (fv_nphys > 0) ", but specifically:
112-142
300-328
468-487

Reply:
Ok great, thanks for helping focus the lines for this file.  I'm nearly finished with my review, now I need to finish dyn_grid.F90 which similarly has a significant number of changes so it may take some time.

Reply:
yea, you'll definitely need to take your time through that file. That's where the most important changes happened.